felix/bpf-gpl/conntrack_types.h

```diff
@@ -120,12 +123,14 @@ static CALI_BPF_INLINE void __xxx_compile_asserts(void) {
 #define ct_value_set_flags(v, f) do {		\
 	(v)->flags |= ((f) & 0xff);		\
 	(v)->flags2 |= (((f) >> 8) & 0xff);	\
+	(v)->flags3 |= (((f) >> 16) & 0xff);	\
+	(v)->flags4 |= (((f) >> 24) & 0xff);	\
 } while(0)
 
-#define ct_value_get_flags(v) ({			\
-	__u16 ret = (v)->flags | ((v)->flags2 << 8);	\
-							\
-	ret;						\
+#define ct_value_get_flags(v) ({									\
```

Comment:
is your editor converting tabs to spaces?

Reply:
In this case, it was the opposite... comments were aligned with spaces. I converted to tabs - it looks aligned with tabsize == 8spaces, but Github seems to visualise tabs with 4 spaces 🤔 Can return it to how it was if you like.